middleware/source_ips.go

```diff
@@ -72,7 +78,15 @@ func extractHost(address string) string {
 
 // Get returns any source addresses we can find in the request, comma-separated
 func (sips SourceIPExtractor) Get(req *http.Request) string {
-	fwd := extractHost(sips.getIP(req))
+	hosts := []string{}
+
+	// Remove port informations from extracted address
+	for _, addr := range sips.getIP(req) {
+		hosts = append(hosts, extractHost(addr))
+	}
+
+	fwd := strings.Join(hosts, ", ")
```

Comment:
nit: Should we prefer `strings.Join(hosts, ",")` so that consumers don't have to trim whitespace?

Reply:
Change made

Reply:
That would change the current behavior as in current implementation this space is present. However if it's ok with you we can change it ?

Reply:
Nah, good point. Let's leave it.